test/v3/connectors/SubmitSelfEmploymentBsasConnectorSpec.scala

```diff
@@ -37,36 +37,37 @@ class SubmitSelfEmploymentBsasConnectorSpec extends ConnectorSpec {
       expenses = Some(expensesModel)
     )
 
-  val nino = Nino("AA123456A")
-  val bsasId = "f2fb30e5-4ab6-4a29-b3c1-c7264259ff1c"
+  val nino: Nino = Nino("AA123456A")
+  val bsasId     = "f2fb30e5-4ab6-4a29-b3c1-c7264259ff1c"
 
-  class Test extends MockHttpClient with MockAppConfig {
+  trait Test extends MockHttpClient with MockAppConfig {
```

Comment:
Can we make this spec consistent with how we've done connector specs elsewhere?

`  trait Test { _: ConnectorTest =>

    val connector: SubmitSelfEmploymentBsasConnector = new SubmitSelfEmploymentBsasConnector(
      http = mockHttpClient,
      appConfig = mockAppConfig
    )

  }`